docs/applications/server.md

```diff
@@ -106,6 +106,10 @@ For more information on how to use our built in `cli` to interact with the serve
     - `message` - message to put into OP_RETURN commitment
     - `--hashMessage` - should the message be hashed before commitment
     - `--feerate <value>` - Fee rate in sats per virtual byte
+ - `lockunspent` `unlock` `transactions` - Temporarily lock (unlock=false) or unlock (unlock=true) specified transaction outputs.
+    - `unlock` - Whether to unlock (true) or lock (false) the specified transactions
+    - `transactions` - The transaction outputs and within each, the txid
```

Comment:
This isn't really clear to me... tx output within each what? Do i also need to submit the txid?

Reply:
Actual code examples go a long way for things like this

Reply:
woops missed part of the description.